Mixpanel/Mixpanel.m

```diff
@@ -838,26 +849,33 @@ - (NSDictionary *)collectAutomaticProperties
                                   @"mp_lib": @"iphone",
                                   @"$lib_version": [self libVersion],
                                   @"$manufacturer": @"Apple",
+#if !defined(MIXPANEL_WATCH_EXTENSION)
```

Comment:
i'd rather have only one `defined` here and not pick properties here and there depending on the target. Have you considered modularizing how we collect the properties in a way that is cleaner to know what properties will be included depending on the target?